_docs/reference/release-notes.md

```diff
@@ -8,4 +8,17 @@ layout: docs
 type: markdown
 ---
 
-TBD
+## Istio v0.1
+
+Istio v0.1 is the initial release of Istio. It works in a single Kubernetes cluster with a small number of services. Istio v0.1 supports the following features:
```

Comment:
Why small clusters? I suggest removing all the qualifications unless we have numbers to prove one way or another